project/plugins.sbt

```
@@ -21,3 +21,5 @@ addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.6.25")
 addSbtPlugin("org.scoverage" %% "sbt-scoverage" % "1.5.1")
 
 addSbtPlugin("org.scalastyle" %% "scalastyle-sbt-plugin" % "1.0.0")
+
+addSbtPlugin("org.irundaia.sbt" % "sbt-sassify" % "1.4.11")
```

```suggestion
addSbtPlugin("org.irundaia.sbt" % "sbt-sassify" % "1.4.13")
```
Can you up the version of this as suggested?